trailblazer/mip/files.py

```diff
@@ -52,6 +52,7 @@ def parse_sampleinfo(data: dict) -> dict:
         'family': data['family'],
         'genome_build': genome_build_str,
         'rank_model_version': data['program']['genmod']['rank_model']['version'],
+        'sv_rank_model_version': data['program']['genmod']['sv_rank_model']['version'],
```

Comment:
```suggestion
        'sv_rank_model_version': data['program']['sv_genmod']['sv_rank_model']['version'],
```
As per @emiliaol's suggestion!